iceberg/iceberg-handler/src/test/results/positive/vectorized_iceberg_read.q.out

```diff
@@ -129,17 +129,17 @@ Stage-0
     Stage-1
       Reducer 2 vectorized
       File Output Operator [FS_11]
-        Select Operator [SEL_10] (rows=1 width=564)
+        Select Operator [SEL_10] (rows=1 width=372)
```

Comment:
Out of curiosity: do you know why the width has descreased so much?

Reply:
TBH I am not sure, but I expect that has something to do with the new statistics